src/pages/version/ParametersModal/index.tsx

```diff
@@ -26,23 +35,34 @@ export const ParametersModal: React.FC<ParametersProps> = ({ parameters }) => {
         </MetadataItem>
       )}
       <DisplayModal
+        data-cy="parameters-modal"
         open={showModal}
         setOpen={setShowModal}
+        size="large"
```

Comment:
I kinda feel like this modal is super wide considering ime these values are often fairly short. However if you have seen examples otherwise I think large is fine!

Reply:
ah yes they can be really long! I've seen a patch parameter that was a really long URL